source/NVDAObjects/window/edit.py

```diff
@@ -823,7 +823,6 @@ def event_caret(self):
 		if eventHandler.isPendingEvents('valueChange',self):
 			self.hasContentChangedSinceLastSelection=True
 		super(Edit,self).event_caret()
-		self.detectPossibleSelectionChange()

```

Comment:
For justification of this removal, see https://github.com/nvaccess/nvda/issues/5770#issuecomment-218611354